pkg/render/compliance.go

```diff
@@ -325,23 +330,28 @@ func (c *complianceComponent) complianceControllerClusterRole() *rbacv1.ClusterR
 				Resources: []string{"globalreports/finalizers"},
 				Verbs:     []string{"update"},
 			},
+			{
+				APIGroups: []string{"linseed.tigera.io"},
+				Resources: []string{"compliancereports"},
+				Verbs:     []string{"create"},
```

Comment:
I do not remember if the controller also needed access to get the latest generated report.